src/app-layout/split-panel/use-split-panel-forced-position.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRef } from 'react';
+
+import { SPLIT_PANEL_MIN_WIDTH } from './constants';
+
+// Magnitude which should be equal or larger than the browser scrollbar thickness
+const scrollbarThreshold = 20;
```

Comment:
Let's calculate the actual scroll width and use this value as a fallback

Reply:
Done in latest revision, thank you for the suggestion. Ran through the Lighthouse tests in my pipeline to make sure there was no performance regression.

The existing `browserScrollbarSize` function already takes care of memoizing the value by using the `calculateOnce` function.